app/validators/questions/mutually_exclusive_validator.py

```diff
@@ -6,14 +6,19 @@ class MutuallyExclusiveQuestionValidator(QuestionValidator):
     MUTUALLY_EXCLUSIVE_CONTAINS_MANDATORY = (
         "MutuallyExclusive question type cannot contain mandatory answers."
     )
-    NON_CHECKBOX_ANSWER = "Question is not of type Checkbox."
+    NON_CHECKBOX_RADIO_ANSWER = "Question is not of type Checkbox or Radio."
 
     def validate(self):
         super().validate()
 
         if any(answer["mandatory"] is True for answer in self.answers):
             self.add_error(self.MUTUALLY_EXCLUSIVE_CONTAINS_MANDATORY)
 
-        if self.answers[-1]["type"] != "Checkbox":
-            self.add_error(self.NON_CHECKBOX_ANSWER, answer_id=self.answers[-1]["id"])
+        if (
+            self.answers[-1]["type"] != "Checkbox"
+            and self.answers[-1]["type"] != "Radio"
```

Comment:
Now that we allow radios. We should validate that the other answers isn't a Radio. I would loop though all answers that isn't the last one, as soon we will have multiple answers that aren't the exclusive answer.

Reply:
> Now that we allow radios. We should validate that the other answers isn't a Radio. I would loop though all answers that isn't the last one, as soon we will have multiple answers that aren't the exclusive answer.

Is this needed? Because the DS doesn't allow radios to have mutually exclusive options

Reply:
Validator is stop invalid schemas. Because the DS doesn't allow it, is the reason why we need to validate, otherwise it would error at run time.